### PR TITLE
[css-scroll-snap] scroll-snap-type computed value

### PR DESCRIPTION
**Pull request by ewilligers**
Spec: https://drafts.csswg.org/css-scroll-snap-1/

Blink/Safari fail by reporting the scroll-snap-type computed value
as "x proximity" instead of "x".

### CODE REVIEW

**Comment by ewilligers** [comment]
Note the following two principles in [CSSOM](https://drafts.csswg.org/cssom/#serializing-css-values):

> If component values can be omitted or replaced with a shorter representation without changing the meaning of the value, omit/replace them.

> If either of the above syntactic translations would be less backwards-compatible, do not perform them.
 
Blink and Safari are the only browsers that support scroll-snap-type values like "x" and "x proximity", and they give "x proximity" as the computed value, so expecting "x proximity" would be more backwards-compatible.

**Comment by ewilligers** [comment]
@tabatkins please review

File: css/css-scroll-snap/parsing/scroll-snap-align-computed.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<body>
+<div id="target"></div>
+<script>
+test_computed_value("scroll-snap-align", "none");
+test_computed_value("scroll-snap-align", "start");
+test_computed_value("scroll-snap-align", "end");
+test_computed_value("scroll-snap-align", "center");
+
+test_computed_value("scroll-snap-align", "start none");
+test_computed_value("scroll-snap-align", "center end");
```
Comment (tabatkins) [inline css/css-scroll-snap/parsing/scroll-snap-align-computed.html]:
```suggestion
test_computed_value("scroll-snap-align", "center end");
test_computed_value("scroll-snap-align", "start start", "start");
```

**Review by tabatkins — CHANGES_REQUESTED** [review]

**Comment by tabatkins** [comment]
> Blink and Safari are the only browsers that support scroll-snap-type values like "x" and "x proximity", and they give "x proximity" as the computed value, so expecting "x proximity" would be more backwards-compatible.

Back-compat is about pages depending on the behavior, not browsers happening to have done it a particular way in the past. (That's colloquially called "bugwards compat". ^_^) This is new enough that I doubt anyone's depending on our too-long serializations, so we should continue to follow shortest-serialization principle until shown evidence otherwise.

**Review by tabatkins — APPROVED** [review]

**Pull request merged**